packages/react/src/components/Tree/TreeItem.tsx

```diff
@@ -117,7 +117,7 @@ class TreeItem extends UIComponent<WithAsProp<TreeItemProps>, TreeItemState> {
     onFocusFirstChild: PropTypes.func,
     onFocusParent: PropTypes.func,
     onSiblingsExpand: PropTypes.func,
-    open: PropTypes.bool,
+    expanded: PropTypes.bool,
```

Comment:
I think this should be treated as auto controlled prop, by defining `defaultExtended` and `onExtendedChange`. With this we will allow the user to define only the initial state, if they want that, or listen to the `onExtendedChange` and change the `extended` prop by themselves.

Reply:
we are not having any state in TreeItem. All expanded stateful logic is in Tree, since TreeItems can be unmounted on virtualization. This is just a prop. The user can control it as it is.

Reply:
I am not sure about this one. I understand that it is just a prop, but all other similar props are treated as autocontrolled. @miroslavstastny thoughts on this?